src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst

```diff
@@ -753,7 +753,8 @@ Example with the ``>`` operator:
 
 Category
 ********
-:Apply: apply on 'categories' field
+:Apply: 'keyword' datatype
```

Comment:
can you use the same semantic that Samir's?
https://github.com/jjanvier/pim-community-dev/pull/75/files#diff-f46b60971d1e2bc2731d390e2d33050dR905